doc/UserGuide.md

```diff
@@ -66,6 +66,13 @@ Examples:
 * `find Betsy Tim John`<br>
   Returns Any person having names `Betsy`, `Tim`, or `John`
 
+### Finding all persons containing any keyword in their tags: `findtags`
+Finds persons whose tags contains any of the given keywords.<br>
+Format: `findtags KEYWORD [MORE_KEYWORDS]`
+
+> The search is case sensitive, the order of the keywords does not matter, only the tags are searched, 
```

Comment:
Limitations of the case sensitivity clearly stated, would it be possible to change it to case insensitive instead?

Reply:
Thanks for the recommendation! Will implement this